packages/toolkit/src/query/core/buildMiddleware/polling.ts

```diff
@@ -126,17 +127,23 @@ export const buildPollingHandler: InternalHandlerBuilder = ({
   }
 
   function findLowestPollingInterval(subscribers: Subscribers = {}) {
+    let skipPollOnFocusLost: boolean | undefined = false
     let lowestPollingInterval = Number.POSITIVE_INFINITY
     for (let key in subscribers) {
       if (!!subscribers[key].pollingInterval) {
         lowestPollingInterval = Math.min(
           subscribers[key].pollingInterval!,
           lowestPollingInterval
         )
+        skipPollOnFocusLost = subscribers[key].skipPollOnFocusLost
```

Comment:
unsure of what the intent here is - currently this just means that skipPollOnFocusLost will be whatever the last subscription in the map has it set to.

if the desired approach is to skip if at least one of the subscriptions says to, you'd need something like 
```suggestion
        skipPollOnFocusLost = subscribers[key].skipPollOnFocusLost || skipPollOnFocusLost
```

Reply:
Yeah absolutely was not sure what the right way to handle that behavior was and don't have a very deep understanding of the subscription logic. So I just chose to use the last subscription and wrote that intent in the test behavior, but I can see how this is better.

Most recent commit should reflect this change + test and test name

Reply:
looks good, thanks!